CGATPipelines/Pipeline/Execution.py

```diff
@@ -368,8 +368,8 @@ def run(**kwargs):
     options.update(kwargs.items())
 
     # insert a few legacy synonyms
-    options['cluster_options'] = options.get('job_options',
-                                             options['cluster_options'])
+    options['cluster_options'] += " " + options.get('job_options',
+                                                    "")
```

Comment:
On the other hand, have you tried calling `Pipeline.submit` with `job_options="-P gen2reg -l h_rt=168:00:00"`

That should give you the result you expect.